lib/generators/rolemodel/mailers/mailers_generator.rb

```diff
@@ -0,0 +1,62 @@
+require_relative '../../bundler_helpers'
+
+
+module Rolemodel
+  class MailersGenerator < Rails::Generators::Base
+    include Rolemodel::BundlerHelpers
+    source_root File.expand_path('templates', __dir__)
+
+    def install_premailer_rails
+      gem 'premailer-rails'
+      run_bundle
+    end
+
+    def add_premailer_rails_config
+      copy_file 'config/initializers/premailer_rails.rb'
+    end
+
+    def add_action_mailer_asset_host
+      unless File.exist?(Rails.root.join('config/initializers/devise.rb'))
+        inject_into_file 'config/environments/development.rb', after: "config.action_mailer.perform_caching = false\n" do
+          optimize_indentation <<~'RUBY', 2
+
+            # Default mailing host suggested by Devise installation instructions
+            config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+          RUBY
+        end
+      end
+      
+      inject_into_file 'config/environments/development.rb', after: "config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }\n" do
+        optimize_indentation <<~'RUBY', 2
+
+          # Ensure premailer_rails can point to webpack compiled resources
+          # https://github.com/fphilipe/premailer-rails/issues/232#issuecomment-839819705
+          config.action_mailer.asset_host = 'http://localhost:3000'
+        RUBY
+      end
+    end
+
+    def add_mailer_css
+      copy_file 'app/javascript/packs/mailer_stylesheets.scss'
+    end
+
+    def add_mailer_layout
+      copy_file 'app/views/layouts/mailer.html.slim'
+    end
+
+    def add_mailer_template_logo
+      copy_file 'public/logo.png'
```

Comment:
Wondering if this should be the RoleModel logo or some kind of other placeholder image

Reply:
We have it as the RMS logo for now and I would assume each project would pull in their own logo to replace it. I could come up with a generic logo instead.